Automattic/jetpack
# Integrate Pexels to Gutenberg media sidebar

Part of https://github.com/Automattic/wp-calypso/issues/71232

## Proposed changes:

* Added Pexels to Gutenberg's Media Sidebar

### Pexels image selector
![Captura de Pantalla 2023-07-18 a las 15 07 11](https://github.com/Automattic/jetpack/assets/1989914/723aec6e-d6a1-45aa-9992-b9e1fc4285a6)


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to a Jetpack-connected site.
* Edit or create a new Post.
* Open the Media Sidebar
* You should see the Pexels integration.
